x-pack/plugins/ml/public/alerting/jobs_health_rule/tests_selection_control.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useCallback } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiFormFieldset, EuiSpacer, EuiSwitch } from '@elastic/eui';
+import { JobsHealthRuleTestsConfig } from '../../../common/types/alerts';
+import { getResultJobsHealthRuleConfig } from '../../../common/util/alerts';
+
+interface TestsSelectionControlProps {
+  config: JobsHealthRuleTestsConfig;
+  onChange: (update: JobsHealthRuleTestsConfig) => void;
+}
+
+export const TestsSelectionControl: FC<TestsSelectionControlProps> = ({ config, onChange }) => {
+  const uiConfig = getResultJobsHealthRuleConfig(config);
+
+  const updateCallback = useCallback(
+    (update: Partial<Exclude<JobsHealthRuleTestsConfig, undefined>>) => {
+      onChange({
+        ...(config ?? {}),
+        ...update,
+      });
+    },
+    [onChange, config]
+  );
+
+  return (
+    <EuiFormFieldset
+      legend={{
+        children: i18n.translate(
+          'xpack.ml.alertTypes.jobsHealthAlertingRule.testsSelection.legend',
+          {
+            defaultMessage: 'Select health checks to perform',
+          }
+        ),
+      }}
+    >
+      <EuiSwitch
+        label={
+          <FormattedMessage
+            id="xpack.ml.alertTypes.jobsHealthAlertingRule.testsSelection.datafeedCheck.label"
+            defaultMessage="Datefeed is not started"
+          />
+        }
+        onChange={updateCallback.bind(null, { dataFeed: { enabled: !uiConfig.dataFeed.enabled } })}
+        checked={uiConfig.dataFeed.enabled}
+      />
+
+      <EuiSpacer size="s" />
+
+      <EuiSwitch
+        label={
+          <FormattedMessage
+            id="xpack.ml.alertTypes.jobsHealthAlertingRule.testsSelection.mmlCheck.label"
+            defaultMessage="Model memory limit monitoring"
+          />
+        }
+        onChange={updateCallback.bind(null, { mml: { enabled: !uiConfig.mml.enabled } })}
+        checked={uiConfig.mml.enabled}
+      />
+
+      <EuiSpacer size="s" />
+
+      <EuiSwitch
+        label={
+          <FormattedMessage
+            id="xpack.ml.alertTypes.jobsHealthAlertingRule.testsSelection.delayedDataCheck.label"
+            defaultMessage="Delayed data has occurred"
```

Comment:
Just `Delayed data` might be simpler here

Reply:
Fixed in https://github.com/elastic/kibana/pull/106084/commits/29f547a3c6924bf1a22249b499584da00f474810